test/components/views/dialogs/ForwardDialog-test.tsx

```diff
@@ -75,20 +76,15 @@ describe("ForwardDialog", () => {
         mockClient.getVisibleRooms.mockReturnValue(rooms);
         mockClient.getRoom.mockImplementation(roomId => rooms.find(room => room.roomId === roomId));
 
-        let wrapper;
-        await act(async () => {
-            wrapper = mount(
-                <ForwardDialog
-                    matrixClient={mockClient}
-                    event={message}
-                    permalinkCreator={new RoomPermalinkCreator(undefined, sourceRoom)}
-                    onFinished={jest.fn()}
-                />,
-            );
-            // Wait one tick for our profile data to load so the state update happens within act
-            await new Promise(resolve => setImmediate(resolve));
-        });
-        wrapper.update();
+        const wrapper: RenderResult = render(
+            <ForwardDialog
+                matrixClient={mockClient}
+                event={message}
+                permalinkCreator={new RoomPermalinkCreator(undefined, sourceRoom)}
+                onFinished={jest.fn()}
+            />,
+        );
+        await new Promise(resolve => setImmediate(resolve));
```

Comment:
Can we get rid of this and instead use `findX` or `waitFor` queries which do magical retrying?